app/controllers/users/two_factor_authentication_setup_controller.rb

```diff
@@ -7,15 +7,14 @@ class TwoFactorAuthenticationSetupController < ApplicationController
     before_action :authenticate_user
 
     def index
-      @user_phone_form = UserPhoneForm.new(current_user)
+      @two_factor_options_form = TwoFactorOptionsForm.new(current_user)
       analytics.track_event(Analytics::USER_REGISTRATION_PHONE_SETUP_VISIT)
     end
 
-    def set
-      @user_phone_form = UserPhoneForm.new(current_user)
-      result = @user_phone_form.submit(params[:user_phone_form])
-
-      analytics.track_event(Analytics::MULTI_FACTOR_AUTH_PHONE_SETUP, result.to_h)
+    def create
+      @two_factor_options_form = TwoFactorOptionsForm.new(current_user)
+      result = @two_factor_options_form.submit(params[:two_factor_options_form])
+      # analytics.track_event(Analytics::USER_REGISTRATION_PHONE_SETUP_VISIT)
```

Comment:
We have some dead code here we may want to revisit. We're going to need to make sure we are tracking the analytics event here.

Reply:
Related, we'll probably want to add a controller spec to make sure this is getting tracked appropriately.